schema.sql

```diff
@@ -8,4 +8,20 @@ CREATE TABLE animals (
     neutered BOOLEAN NOT NULL,
     weight_kg DECIMAL NOT NULL,
 );
-ALTER TABLE animals ADD species VARCHAR(20);
\ No newline at end of file
+ALTER TABLE animals ADD species VARCHAR(20);
+
```

Comment:
In this milestone, the SQL that modifies the animal's table should REMOVE the column species not ADD it.

Reply:
Hi  @KDlamini, Thank you so much for reminding me.